src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -143,9 +138,9 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
         }
 
         private async Task<Document> RemoveSortUsingsAsync(
-            Document document, DocumentOptionSet docOptions, CancellationToken cancellationToken)
+            Document document, OrganizeUsingsSet organizeUsingsSet, CancellationToken cancellationToken)
```

Comment:
i don't really unerstand OrganizeUsingsSet vs DiagnosticsSet... feels like we shoudl be unifying around a consistent way of describing these things.

Reply:
This has been one of my issues from the first round of reviews. In most cases, fixers run against known diagnostic IDs. However, we don't have good diagnostic IDs for the fixers we can apply to using directives (even if #29704 is merged, we can't distinguish between sort and remove unused).

If this isn't cleaned up before merging, it will get a new issue filed for https://github.com/dotnet/roslyn/pull/30039#discussion_r221997210, so I'll resolve this review comment and we can continue the discussion there if needed.